plone/recipe/varnish/recipe.py

```diff
@@ -79,6 +80,23 @@ def __init__(self, buildout, name, options):
         self.options.setdefault("jobs", "4")
         CMMIRecipe.__init__(self, buildout, name, self.options)
 
+    def build(self):
+        super(BuildRecipe, self).build()
```

Comment:
Is it by intend to first call super build here and again in the conditional block? 
If so please add a comment why, because it is not obvious

Reply:
+1

Reply:
Agree... reimplemented in a more clear manner.